### PR TITLE
CHOP-15. Improved exception message when Kiji dependency jars can't be found.

### PR DESCRIPTION
**Pull request by robotoer**
@renuka @lockoff.

### CODE REVIEW

**Comment by renuka-apte** [comment]
Looks good.

File: src/main/java/org/kiji/lang/KijiTap.java [hunk 1]
```diff
@@ -109,7 +109,12 @@ public void sourceConfInit(FlowProcess<JobConf> process, JobConf conf) {
           "tmpjars",
           StringUtils.join(findKijiJars(conf), ","));
     } catch (IOException ioe) {
-      throw new RuntimeException(ioe);
+      throw new RuntimeException(
+          "Chopsticks was unable to find dependency jars for Kiji. If you are using KijiBento "
+          + "make sure you have run the 'kiji-env.sh' script. If not, make sure your KijiSchema "
```
Comment (lockoff) [inline src/main/java/org/kiji/lang/KijiTap.java:114]: run -> sourced

File: src/main/java/org/kiji/lang/KijiTap.java [hunk 2]
```diff
@@ -109,7 +109,12 @@ public void sourceConfInit(FlowProcess<JobConf> process, JobConf conf) {
...
+          + "and KijiMapReduce installations have 'lib' directories containing their "
+          + "dependency jars.",
```
Comment (lockoff) [inline src/main/java/org/kiji/lang/KijiTap.java:116]: The most likely problem is that the user's environment variables KIJI_HOME and KIJI_MR_HOME are not set correctly.  It's less likely that their lib directories will have disappeared.  We should adjust this message to reflect this.

File: src/main/java/org/kiji/lang/KijiTap.java [hunk 3]
```diff
@@ -138,7 +143,12 @@ public void sinkConfInit(FlowProcess<JobConf> process, JobConf conf) {
           "tmpjars",
           StringUtils.join(findKijiJars(conf), ","));
     } catch (IOException ioe) {
-      throw new RuntimeException(ioe);
+      throw new RuntimeException(
+          "Chopsticks was unable to find dependency jars for Kiji. If you are using KijiBento "
```
Comment (lockoff) [inline src/main/java/org/kiji/lang/KijiTap.java:147]: Same notes here.

**Comment by lockoff** [comment]
Thanks @robotoer . I made some notes on the comments that I'm happy to implement before pushing. If that works for you, let me know and I'm on it.

**Comment by robotoer** [comment]
@lockoff Go for it. Thanks!

**Comment by lockoff** [comment]
I'm also going to move the error message into a constant string, since it's identical in both locations.

**Pull request closed without merging**